cli/uninstall/uninstall.go

```diff
@@ -112,12 +112,30 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 			return err
 		}
 	}
+	// We need to try to delete all versions with 'v' prefix before x.y.z version
+	// and without it.
+	versionsToDelete := make([]string, 0, 2)
+	versionsToDelete = append(versionsToDelete, programVersion)
+
+	if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] != 'v' {
+		versionsToDelete = append(versionsToDelete, "v"+programVersion)
+	} else if program == search.ProgramTt &&
+		strings.Contains(programVersion, ".") && programVersion[0] == 'v' {
+		versionsToDelete = append(versionsToDelete, programVersion[1:])
+	}
```

Comment:
Nit, tt install does not have such logic.
```suggestion
	}
```

Reply:
Then it won't work for `tt uninstall tt v0.1.0` when file `tt_0.1.0` will exists.